src/BloomExe/Publish/BloomPub/PublishToBloomPubApi.cs

```diff
@@ -98,12 +98,13 @@ public void RegisterWithApiHandler(BloomApiHandler apiHandler)
 			apiHandler.RegisterEndpointHandler(kApiUrlPart + "usb/start", request =>
 			{
 #if !__MonoCS__
-
-				SetState("UsbStarted");
+				SetState("UpdatingPreview");
+				_publishApi.UpdatePreviewIfNeeded(request);
```

Comment:
It looks to me like this line is duplicated?

Reply:
whoops, fixed